packages/router/tests/acceptance/lazy-routes-test.js

```diff
@@ -6,65 +6,92 @@ import { getGlobalConfig } from '@embroider/macros';
 
 /* global requirejs */
 
-module('Acceptance | lazy routes', function(hooks) {
+module('Acceptance | lazy routes', function (hooks) {
   setupApplicationTest(hooks);
 
   function hasController(routeName) {
-    return Boolean(requirejs.entries[`${ENV.modulePrefix}/controllers/${routeName}`]);
+    return Boolean(
+      requirejs.entries[`${ENV.modulePrefix}/controllers/${routeName}`]
+    );
   }
 
   function hasRoute(routeName) {
-    return Boolean(requirejs.entries[`${ENV.modulePrefix}/routes/${routeName}`]);
+    return Boolean(
+      requirejs.entries[`${ENV.modulePrefix}/routes/${routeName}`]
+    );
   }
 
   function hasTemplate(routeName) {
-    return Boolean(requirejs.entries[`${ENV.modulePrefix}/templates/${routeName}`]);
+    return Boolean(
+      requirejs.entries[`${ENV.modulePrefix}/templates/${routeName}`]
+    );
   }
 
-  function hasComponentTemplate(name) {
-    return Boolean(requirejs.entries[`${ENV.modulePrefix}/templates/components/${name}`]);
+  function hasComponent(name) {
+    return Boolean(requirejs.entries[`${ENV.modulePrefix}/components/${name}`]);
```

Comment:
@simonihmig I updated the path and renamed the function. Should be good now! :pray: